### PR TITLE
Add Fyde App for Linux install steps

### PR DESCRIPTION
**Pull request by GMartinez-Sisti**
Added:
- Debian/Ubuntu
- CentOS

We should only merge this after the official releases (@rasmartins).

### CODE REVIEW

File: fyde-app/linux.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+parent: Fyde App
+nav_order: 1
+---
+# Fyde App - Linux
+
+Instructions to install the Fyde App in different distributions.
+
+There are two versions available:
+
+- fyde - complete GUI version
```
Comment (oNaiPs) [inline fyde-app/linux.md]: @GMartinez-Sisti perhaps you can refer here (or somewhere below) that `fyde` includes `fyde-cli`

Comment (GMartinez-Sisti) [reply fyde-app/linux.md]: @oNaiPs You mean in the repository?

Comment (oNaiPs) [reply fyde-app/linux.md]: @GMartinez-Sisti apologies, I meant `fyde-tunnel`... My point is just for the users to know that if they install `fyde`, they dont need to install `fyde-tunnel` to have the daemon only interface

Comment (GMartinez-Sisti) [reply fyde-app/linux.md]: Makes sense @oNaiPs

Comment (GMartinez-Sisti) [reply fyde-app/linux.md]: Updated.

File: fyde-app/linux.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+nav_order: 1
+---
+# Fyde App - Linux
+
+Instructions to install the Fyde App in different distributions.
+
+There are two versions available:
+
+- fyde - complete GUI version (includes fyde-tunnel)
+- fyde-tunnel - CLI version
```
Comment (rasmartins) [inline fyde-app/linux.md]: I would write something like this:

Fyde App for Linux is distributed in two separate packages:

- fyde - GUI for interactive user sessions (depends on fyde-tunnel)
- fyde-tunnel - standalone network services and command line utilities designed to run in headless setups

File: fyde-app/linux.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+Supported versions:
+
+| Distro    | Versions      | Instructions              |
+| ---       | ---           | ---                       |
+| Arch      | -             | In progress               |
+| CentOS    | 7, 8          | [YUM](##yum-repository)   |
+| Debian    | 9, 10         | [APT](##apt-repository)   |
+| Linux     | x86_64        | In progress               |
```
Comment (rasmartins) [inline fyde-app/linux.md]: I would write "Other Distribution" and place it in the end of the table.

File: fyde-app/linux.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+        ```sh
+        sudo apt install fyde-tunnel
+        ```
+
+        - Enroll
+
+        ```sh
+        /opt/fyde/fyde-tunnel --enroll "<Fyde App enrollment link>"
+        sudo systemctl --system restart fyde-interface
```
Comment (rasmartins) [inline fyde-app/linux.md]: This is not needed, it is started when the package fyde-tunnel is installed.

File: fyde-app/linux.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+        ```sh
+        sudo yum install fyde-tunnel
+        ```
+
+        - Enroll
+
+        ```sh
+        /opt/fyde/fyde-tunnel --enroll "<Fyde App enrollment link>"
+        sudo systemctl --system restart fyde-interface
```
Comment (rasmartins) [inline fyde-app/linux.md]: This is not needed.

File: fyde-app/linux.md (hunk 2 above)
Comment (GMartinez-Sisti) [reply fyde-app/linux.md]: That's better indeed. Going to change.

File: fyde-app/linux.md (hunk 4 above)
Comment (GMartinez-Sisti) [reply fyde-app/linux.md]: Got it. Removed.

File: fyde-app/linux.md (hunk 5 above)
Comment (GMartinez-Sisti) [reply fyde-app/linux.md]: Got it.

**Comment by GMartinez-Sisti** [comment]
@rasmartins All your suggestions were applied. Please check.

Thanks

File: fyde-app/linux.md (hunk 3 above)
Comment (rasmartins) [reply fyde-app/linux.md]: What about this change @GMartinez-Sisti

Comment (GMartinez-Sisti) [reply fyde-app/linux.md]: That's what you get for multitasking. Updated.

**Pull request merged**